cmake/IgnUtils.cmake

```diff
@@ -602,6 +602,7 @@ endfunction()
 #   [EXCLUDE_FILES <excluded_headers>]
 #   [EXCLUDE_DIRS  <dirs>]
 #   [GENERATED_HEADERS <headers>]
+#   [INSTALL_BUT_DONT_INCLUDE <headers>]
```

Comment:
this parameter name is confusing to me

an alternative approach is to use `EXCLUDE_FILES` and then manually install the desired headers, as we don in sdformat:

* https://github.com/ignitionrobotics/sdformat/blob/sdf12/include/sdf/CMakeLists.txt#L1-L8

it could benefit from an easier install helper so it doesn't need to pass `${IGN_INCLUDE_INSTALL_DIR_FULL}/${PROJECT_INCLUDE_DIR}` as the destination

`ign_install_includes` will provide the first half of that path, but you'd still have to pass `${PROJECT_INCLUDE_DIR}`

Reply:
Ahh much better, thanks!